doc/source/tutorials/bipartite_matching_maxflow/assets/bipartite_matching_maxflow.py

```diff
@@ -0,0 +1,51 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph(
+    9,
+    [(0, 4), (0, 5), (1, 4), (1, 6), (1, 7), (2, 5), (2, 7), (2, 8), (3, 6), (3, 7)],
+    directed=True
+)
+
+# Assign nodes 0-3 to one side, and the nodes 4-8 to the other side
+for i in range(4):
+    g.vs[i]["type"] = True
+for i in range(4, 9):
+    g.vs[i]["type"] = False
+
+# Add source and sink as nodes 9 and 10
+g.add_vertices(2)
+g.add_edges([(9, 0), (9, 1), (9, 2), (9, 3)]) # connect source to one side
+g.add_edges([(4, 10), (5, 10), (6, 10), (7, 10), (8, 10)]) # ... and sinks to the other
+
+flow = g.maxflow(9, 10)
+print("Size of Maximum Matching (maxflow) is:", flow.value)
+
+# Compare this to the "maximum_bipartite_matching()" function
+g2 = g.copy()
+g2.delete_vertices([9, 10]) # delete the source and sink, which are unneeded for this function.
+matching = g2.maximum_bipartite_matching()
+
+matching_size = 0
+for i in range(4):
+    if matching.match_of(i):
+        matching_size += 1
```

Comment:
Probably a more pythonic way to say this is `matching_size = sum(1 for i in range(4) if matching.match_of(i) is not None)`. Again, watch out for the potential case of `matching.match_of(i) == 0`.